installation/install.sh

```
@@ -2488,6 +2497,10 @@ install_db(){
     /bin/rm -rf $ssh_tmp_dir
     mkdir -p $ssh_tmp_dir
     show_spinner cs_gen_sshkey $ssh_tmp_dir
+    #ensure sshd is running before ansible connects via ssh (alinux4 may not have sshd started at firstboot)
+    if [ x"$OS" = x"ALINUX4" ]; then
```

Shellcheck (0.11.0)

[style] Avoid x-prefix in comparisons as it no longer serves a purpose.

(SC2268)